Python/compile.c

```diff
@@ -235,6 +235,28 @@ _PyCompile_InstructionSequence_UseLabel(instr_sequence *seq, int lbl)
     return SUCCESS;
 }
 
+int
+_PyCompile_InstructionSequence_ApplyLabelMap(instr_sequence *instrs)
```

Comment:
Maybe just `_PyCompile_ApplyLabelMap`? 
The `InstructionSequence` seems redundant given the argument is an instruction sequence.
You choose.

Reply:
I'm planning to move the instruction-sequence data structure implementation out of compile.c, so it will probably end up being _PyInstructionSequence_ApplyLabelMap.